src/components/Routes.js

```diff
@@ -18,6 +19,7 @@ const Routes = () => (
       }}
     />
     <Route exact path="/" component={Welcome} />
+    <Route exact path="/landing" component={LandingPage} />
```

Comment:
The landing page is what the user sees before visiting any route so it should have empty path ie "/" for our case it will replace the welcome

Reply:
I worked on the feedback